app/src/lib/app-shell.ts

```diff
@@ -35,7 +35,28 @@ export interface IAppShell {
 }
 
 export const shell: IAppShell = {
-  moveItemToTrash: electronShell.trashItem,
+  // Since Electron 13, shell.trashItem doesn't work from the renderer process
+  // on Windows. Therefore, we must invoke it from the main process. See
+  // https://github.com/electron/electron/issues/29598
+  moveItemToTrash: path => {
+    return new Promise<void>((resolve, reject) => {
+      ipcRenderer.once(
+        'move-to-trash-result',
+        (
+          event: Electron.IpcRendererEvent,
+          { error }: { error: Error | null }
+        ) => {
+          if (error === null) {
+            resolve()
+          } else {
+            reject(error)
+          }
+        }
+      )
+
+      ipcRenderer.send('move-to-trash', { path })
```

Comment:
Could we use [ipcRenderer.invoke](https://www.electronjs.org/docs/latest/api/ipc-renderer#ipcrendererinvokechannel-args) instead of send here to avoid setting up our own single use listener? Seems like the this approach could suffer problems should we happen to call it before the previous invocation completed.

Reply:
Absolutely! FWIW I used `.send` because (1) I had no idea about that and (2) the rest of the methods below in this very same file also use `.send`. Should we change all of those too? (If not now, at some point)

Reply:
> the rest of the methods below in this very same file also use .send. Should we change all of those too? (If not now, at some point)

Send is fine when you don't need a result and the operation can't fail (or we don't care that it fails) but yeah, we should definitely change that `openExternal` to use `invoke` instead. I imagine `invoke` didn't exist at the time that was written or we just didn't catch it.

Reply:
@niik fixed in 6ea87e0bb09ab2ec8c73f78b468d24427ef9fc1d

Took me a while to rebuild and retest everything, sorry! But seems to work now on macOS and Windows 😌